ports-of-call/portability.hpp

```diff
@@ -53,6 +53,7 @@
 #define PORTABLE_MALLOC(size) Kokkos::kokkos_malloc<>(size)
 #define PORTABLE_FREE(ptr) Kokkos::kokkos_free<>(ptr)
 // Do we want to include additional terms here (for memory spaces, etc.)?
+#define PORTABLE_FENCE(name) Kokkos::fence(name)
```

Comment:
Do I have access to Kokkos execution spaces? If so, it would make more sense to fence the device execution space by itself.

Reply:
We don't... I think adding execution spaces generically unfortunately moves us more towards mimicking additional kokkos functionality. Let's leave this as is for now but add an issue?

Reply:
🤷 I'm not sure it even needs an issue since, as you said, that's going a bit beyond what we want to do.